log.go

```diff
@@ -82,7 +82,6 @@ var (
 	ltndLog = addLndPkgLogger("LTND")
 	rpcsLog = addLndPkgLogger("RPCS")
 	srvrLog = addLndPkgLogger("SRVR")
-	fndgLog = addLndPkgLogger("FNDG")
```

Comment:
ok, removed here, nice. Still think it is better to keep the log prefix as before though.